### PR TITLE
Add wasShopperInsightsUsed 👾

### PR DESCRIPTION
**Pull request by spencersablan**
**JIRA:** [DTPPCPSDK-2369](https://paypal.atlassian.net/browse/DTPPCPSDK-2369)

- _**💥 Adds wasShopperInsightsUsed function**_ - Checks session state for key indicators that shopper insights was used

### CODE REVIEW

File: src/session.test.js [hunk 1]
```diff
@@ -75,4 +76,8 @@ describe("session cases", () => {
       stickySessionId: expect.any(String),
     });
   });
+
+  it("wasShopperInsightsUsed should return false if shopperInsights doesn't exist in the state", () => {
```
Comment (wsbrunson) [inline src/session.test.js:80]: we should a true case as well

**Review by wsbrunson — APPROVED** [review]

**Review by nikrom17 — APPROVED** [review]

File: src/session.test.js (hunk 1 above)
Comment (spencersablan) [reply src/session.test.js:80]: Definitely. 👍   I was having some issues with mocking some of the storage functions before my vacation & realized all the other functions only really have negative testing.  But let me see if a fresh brain can come up with something better now 😄

Comment (spencersablan) [reply src/session.test.js:80]: Haha actually, it was pretty easy. See.. fresh brain 🧠

Comment (wsbrunson) [reply src/session.test.js]: fresh brains solve most things

File: src/session.test.js [hunk 2]
```diff
@@ -75,4 +76,59 @@ describe("session cases", () => {
       stickySessionId: expect.any(String),
     });
   });
+
+  describe.only("wasShopperInsightsUsed", () => {
```
Comment (wsbrunson) [inline src/session.test.js:80]:
```suggestion
  describe("wasShopperInsightsUsed", () => {
```

**Review by wsbrunson — APPROVED** [review]

**Pull request merged**